internal/command/deploy/machines.go

```diff
@@ -107,6 +116,62 @@ func (lm *leasableMachine) Update(ctx context.Context, input api.LaunchMachineIn
 	return nil
 }
 
+func (md *machineDeployment) logClearLinesAbove(count int) {
+	if md.io.IsInteractive() {
+		builder := aec.EmptyBuilder
+		str := builder.Up(uint(count)).EraseLine(aec.EraseModes.All).ANSI
+		fmt.Fprint(md.io.ErrOut, str.String())
+	}
+}
+
+func (lm *leasableMachine) logClearLinesAbove(count int) {
+	if lm.io.IsInteractive() {
+		builder := aec.EmptyBuilder
+		str := builder.Up(uint(count)).EraseLine(aec.EraseModes.All).ANSI
+		fmt.Fprint(lm.io.ErrOut, str.String())
+	}
+}
+
+func (lm *leasableMachine) logStatus(desired, current string) {
+	cur := aurora.Green(current)
```

Comment:
Just a note here that people sometimes turn colors off, so we're using `colorize` from the `iostreams` package, which is available on the global context. See [[So about storj, ‘amazingly well’ only for registry, or for storage in general? ](https://github.com/superfly/flyctl/blob/8a55d4067be1d5c7c78890ea7a28ff3016d8a848/internal/machine/restart.go#L32-L35)]